docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md

```diff
@@ -1,27 +1,56 @@
 # CREATE INDEX
 
+Use `CREATE INDEX` to add indexes to existing tags, edge types or properties.
+
+Most graph queries start the traversal from a list of vertices or edges that are identified by their properties. Indexes make these global retrieval operations efficient on large graphs.
+
+## Prerequisites
+
+Before you create an index, make sure that the relative tag or edge type is created. For how to create tags or edge types, see CREATE EDGE (doc TODO) and CREATE EDGE (doc TODO).
+
+## Must-read for using index
```

Comment:
unfortunately, you have to state very clearly that: 

consider which index to use:

1. text-based index (by elastic search)
2. native-index (by Nebula storage)

![image](https://user-images.githubusercontent.com/50101159/100408555-ff8b7a00-30a5-11eb-8edb-e490d4123f62.png)

1. text-based index (by elastic search)

It is designed to full-text search (i.e., string and fix_string type of property in Nebula Graph). And can only be used for the following four searches: prefix, wildcard, regexp, fuzzy.

In this case, you have to **deploy an elastic search cluster** and introduce it to the Nebula Graph.

See xxx-link

2. native-index (by Nebula storage)

It is (almost) the same as the Nebula Graph v1.X index. see hereafter.

Reply:
re-draw this picture to cover this two concerpt.

Reply:
Thank you. I'll do this in another PR. This PR will focus on the `CREATE INDEX` topic.